test/base-fork/TransferAsset.t.sol

```diff
@@ -64,6 +100,18 @@ contract ForeignControllerTransferAssetFailureTests is TransferAssetBaseTest {
 
 }
 
+contract ForeignControllerTransferAssetTransferFailureTest is TransferAssetFailureBaseTest {
+
```

Comment:
Can just do the setUp in here, doesn't need a base contract if its only used in one place

Reply:
changed